xendit/models/creditcard/credit_card.py

```diff
@@ -258,6 +258,7 @@ def create_charge(
         for_user_id=None,
         x_api_version=None,
         metadata=None,
+        with_fee_rule=None,
```

Comment:
[nit] move this above metadata

Reply:
@MESMUR we added with_fee_rule to the last place, because we were concerned that maybe someone uses this method by arguments order.  That's why we do not  want to break someones code.

example:
```create_charge(arg1, arg2, arg3, ...)```

But if you approve, we will move above metadata

Reply:
Ah that's fair, although I would hope implementations of the SDK would use named arguments aha.

Good point, you can leave it as it is!